### PR TITLE
UI update

### LINKED ISSUE #75: Improve the look and feel of the app

### PR DESCRIPTION
**Pull request by MrZaiko**
Fixes #75 #83 

### Done

- UI update of room activity
- UI update of appointment creation
- UI update of calendar activity
- All activities displaying appointment details directly use AppointmentActivity. This reduces code duplication and increases UI coherence. 
- Call : UI support

### Remaining tasks

- [x] Bottom bar AppointmentActivity
- [x] Link ban user UI to database
- [x] Change participants and banned participants UI in DETAIL_MODE in AppointmentActivity
- [x] Refactor AppointmentActivity
- [x] Adapt UI tests
- [x] Javadoc
- [x] Fix Cirrus

### CODE REVIEW

**Review by Fermeli — APPROVED** [review]
Everything looks good to me

File: app/src/main/java/io/github/polysmee/appointments/fragments/DataPasser.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package io.github.polysmee.appointments.fragments;
+
+import java.util.Set;
+
+public interface DataPasser {
```
Comment (MRandl) [inline app/src/main/java/io/github/polysmee/appointments/fragments/DataPasser.java]: hehe what is that

**Review by MRandl — CHANGES_REQUESTED** [review]
Bug (at least from my emulator) : freshly created appointments do not appear on the calendar

File: app/src/main/java/io/github/polysmee/appointments/fragments/DataPasser.java (hunk 1 above)
Comment (MrZaiko) [reply app/src/main/java/io/github/polysmee/appointments/fragments/DataPasser.java]: Ups, forgot the Javadoc. I fix this within the hour

**Comment by MrZaiko** [comment]
> 
> 
> Bug (at least from my emulator) : freshly created appointments do not appear on the calendar

@AttiaYoussef is fixing this.

**Review by AttiaYoussef — APPROVED** [review]

**Review by MRandl — APPROVED** [review]

**Pull request merged**